KIPs/kip-xx.md

```diff
@@ -0,0 +1,52 @@
+---
+kip: to be assigned
+title: Transaction data gas cost reduction
+author: Lake (@hyunsooda) and Ollie (@blukat29)
+discussions-to: https://github.com/kaiachain/kips/issues/22
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-10-18
+---
+
+## Simple Summary
+
+Changing to calldata gas cost.
+
```

Comment:
Because Abstract is already very short, I don't think we need a Simple Summary.

Reply:
Yes, I don't know when but recent EIPs don't have the Simple Summary section. e.g. https://eips.ethereum.org/EIPS/eip-7523